root/multicore/CMakeLists.txt

```diff
@@ -87,15 +87,17 @@ if(ROOT_imt_FOUND)
                     DEPENDS ${GENERATE_EXECUTABLE_TEST})
 endif()
 
-if(NOT MSVC OR win_broken_tests)
-  set(MULTIPROC_LIB "MultiProc")
-endif()
-
-ROOTTEST_GENERATE_EXECUTABLE(tExecutor tExecutor.cxx LIBRARIES Core Imt ${MULTIPROC_LIB} Net TreePlayer RIO Hist MathCore)
+ROOTTEST_GENERATE_EXECUTABLE(tExecutorMT tExecutorMT.cxx LIBRARIES Core Imt Net TreePlayer RIO Hist MathCore)
+ROOTTEST_ADD_TEST(tExecutorMT
+                  EXEC ${CMAKE_CURRENT_BINARY_DIR}/tExecutorMT
+                  DEPENDS ${GENERATE_EXECUTABLE_TEST})
 
-ROOTTEST_ADD_TEST(tExecutor
-                  EXEC ${CMAKE_CURRENT_BINARY_DIR}/tExecutor
+if(NOT MSVG) # TProcessExecutor is not available on windows
```

Comment:
`MSVC`? Not sure if this meant that the test was always executed or never...

Reply:
damn, thanks xD

we want to never execute it on windows -- there is no `fork` there, so no `TProcessExecutor`

Reply:
I get that, but what does the typo do? I would have thought that it makes the test be executed always since `MSVG` is not defined, but Jenkins didn't report an error :thinking:

Reply:
Ah, the PR build doesn't (yet) enable `roottest` on Windows. Never mind then.